common/dpcpp/hashfunctions.hpp

```diff
@@ -59,19 +59,15 @@ struct SimpleHasherWithOffset {
 struct MurmurHash3_x86_32 {
   MurmurHash3_x86_32(size_t sz, int len, uint32_t seed) : _sz(sz), _len(len), _seed(seed) {}
 
-  #define	ROTL32(x,y)	rotl32(x,y)
-
-  #define	FORCE_INLINE inline __attribute__((always_inline))
-
   inline uint32_t rotl32 ( uint32_t x, int8_t r ) const {
     return (x << r) | (x >> (32 - r));
   }
 
-  FORCE_INLINE uint32_t getblock32 ( const uint32_t * p, int i ) const {
+  inline __attribute__((always_inline)) uint32_t getblock32 ( const uint32_t * p, int i ) const {
     return p[i];
   }
 
-  FORCE_INLINE uint32_t fmix32 ( uint32_t h ) const {
+  inline __attribute__((always_inline)) uint32_t fmix32 ( uint32_t h ) const {
```

Comment:
I guess a macro for attributes is fine, less to write, anyway, this is also fine.